pycontrails/models/cocip/cocip.py

```diff
@@ -1274,6 +1279,7 @@ def _fill_empty_flight_results(self, return_list_flight: bool) -> Flight | list[
 def process_met_datasets(
     met: MetDataset,
     rad: MetDataset,
+    compute_tau_cirrus: bool | str = "auto",
     shift_radiation_time: np.timedelta64 | None = None,
```

Comment:
Does it make sense to remove the default values for `compute_tau_cirrus` (and for `shift_radiation_time`)? If we only ever pass model parameters into this function, then we may be less likely to create future bugs if these parameters *don't* take defaults. But if we're calling `process_met_datasets` in places where we don't specify `compute_tau_cirrus` or `shift_radiation_time`, leave as is.

Reply:
The only other place we call `process_met_datasets` is in [tests/benchmark/north-atlantic-study/validate.py](https://github.com/contrailcirrus/pycontrails/blob/9e062cff5ed5f1dc0956e56b6a3ac21958d41405/tests/benchmark/north-atlantic-study/validate.py#L148), is this benchmark test still used? I agree that it would be safer to not include defaults here.

Reply:
We've been meaning to run the benchmark tests occasionally in the github CI. That hasn't happened yet, and so I think there are many slightly broken things in the benchmark directory at this point. No need to worry about that!